shared/js/background/classes/site.es6.js

```diff
@@ -7,21 +7,34 @@
  * privacy score used in the popup.
  */
 const settings = require('../settings.es6')
-const Score = require('./score.es6')
 const utils = require('../utils.es6')
 const abpLists = require('../abp-lists.es6')
+const privacyPolicy = require('../privacy-policy.es6')
+const Grade = require('privacy-grade').Grade
+const trackerPrevalence = require('../../../data/tracker_lists/prevalence')
 
 class Site {
     constructor (domain) {
         if (domain) domain = domain.toLowerCase()
         this.domain = domain
         this.trackerUrls = []
-        this.score = new Score(this.specialDomain(), this.domain)
+        this.score = new Grade()
         this.whitelisted = false // user-whitelisted sites; applies to all privacy features
         this.setWhitelistStatusFromGlobal(domain)
         this.isBroken = this.checkBrokenSites(domain) // broken sites reported to github repo
         this.didIncrementCompaniesData = false
 
+        this.tosdr = privacyPolicy.getTosdr(domain)
+
+        this.parentEntity = utils.findParent(domain) || ''
```

Comment:
Note, the usage for `findParent` changed in https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/244, which got reverted.

I've used it as it was as of #244. So, if we merge this PR before #244 is brought back, **we need to change this line**.